### PR TITLE
Update the README with new information

### PR DESCRIPTION
**Pull request by baxterthehacker**
This is a pretty simple change that we need to pull into master.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# public-repo
```
Comment (baxterthehacker) [inline README.md:1]: Maybe you should use more emojji on this line.

**Comment by robot-muscle** [comment]
Build Failed!
```
/bin/sh: ./ci-build.sh: No such file or directory

```

**Comment by Deborah-Digges** [comment]
Comment

**Comment by Deborah-Digges** [comment]
Comment

**Comment by Deborah-Digges** [comment]
Comment

**Comment by Deborah-Digges** [comment]
Hello @test. Thanks for your contribution! Please have a look at our guide!

**Comment by Deborah-Digges** [comment]
Hello @test. Thanks for your contribution! Please have a look at our contributors guide!

**Comment by Deborah-Digges** [comment]
Thanks for the contibution, @test :penguin: Please do <a href=https://openhatch.readthedocs.org/en/latest/getting_started/handling_contributions.html#permit-us-to-share-your-work>send us an email</a> to permit us to use your work if you haven't done so already.

**Comment by Deborah-Digges** [comment]
Thanks for the contibution, @test :penguin: Please do <a href=https://openhatch.readthedocs.org/en/latest/getting_started/handling_contributions.html#permit-us-to-share-your-work>send us an email</a> to permit us to use your work if you haven't done so already.

**Comment by Deborah-Digges** [comment]
Thanks for the contibution, @test :penguin: Please do <a href=https://openhatch.readthedocs.org/en/latest/getting_started/handling_contributions.html#permit-us-to-share-your-work>send us an email</a> to permit us to use your work if you haven't done so already.

**Comment by Deborah-Digges** [comment]
Thanks for the contibution, @test :penguin: Please do <a href=https://openhatch.readthedocs.org/en/latest/getting_started/handling_contributions.html#permit-us-to-share-your-work>send us an email</a> to permit us to use your work if you haven't done so already.

**Comment by Deborah-Digges** [comment]
Thanks for the contribution, @test :penguin: Please do <a href=https://openhatch.readthedocs.org/en/latest/getting_started/handling_contributions.html#permit-us-to-share-your-work>send us an email</a> to permit us to use your work if you haven't done so already.

**Comment by Deborah-Digges** [comment]
Thanks for the contribution, @test :penguin: Please do <a href=https://openhatch.readthedocs.org/en/latest/getting_started/handling_contributions.html#permit-us-to-share-your-work>send us an email</a> to permit us to use your work if you haven't done so already.

**Comment by Deborah-Digges** [comment]
Thanks for the contribution, @test :penguin: Please do <a href=https://openhatch.readthedocs.org/en/latest/getting_started/handling_contributions.html#permit-us-to-share-your-work>send us an email</a> to permit us to use your work if you haven't done so already.

**Comment by dalanmiller** [comment]
192.168.0.10:5000

**Comment by timminkov** [comment]
Testing please ignore

**Comment by timminkov** [comment]
Your branch currently conflicts with another open pull request: https://api.github.com/repos/teespring/rails-teespring/pulls/8975

**Comment by timminkov** [comment]
Your branch currently conflicts with another open pull request: https://github.com/teespring/rails-teespring/pull/8975

**Comment by allcentury** [comment]
Hello from Node.js

**Comment by oterrien** [comment]
+2

**Comment by sunkibaek** [comment]
2 lines of codes have been changed. (1 additions, 1 deletions) It has changed right amount of codes to review. Thank you.

**Comment by sunkibaek** [comment]
2 lines of codes have been changed. (1 additions, 1 deletions) It has changed right amount of codes to review. Thank you.

**Comment by sunkibaek** [comment]
2 lines of codes have been changed. (1 additions, 1 deletions) It has changed right amount of codes to review. Thank you.

File: README.md (hunk 1 above)
Comment (vincentmg) [reply README.md:1]: Test comment

**Comment by MatthewJWalls** [comment]
Build complete

**Comment by MatthewJWalls** [comment]
Build complete

**Comment by MatthewJWalls** [comment]
Build complete

**Comment by MatthewJWalls** [comment]
Build complete

**Comment by MatthewJWalls** [comment]
Build complete

**Comment by thepont** [comment]
This release can be previewed on staging by deploying
 This will overwrite whatever else is running on staging.
 [[Deploy](undefined/deploy?branch=branch)] [[Server Info](http://staging.toyotainstituteaustralia.com.au/#/admin/site/info)]

**Comment by sampleCrow** [comment]
lilnate22/crowtest.dev.cc.oneplatform.build

**Pull request closed without merging**